blog/2024-01-07-kvrocks-2023-in-review/index.md

```diff
@@ -20,7 +20,7 @@ We collected a lot of feedback from users and developers in the discussion: [202
 - Watch/Unwatch command
 - Lua Functions
 - Replication with TLS
-- 65 bits expire time and size
+- Allow to enable 64 bits expiration time(precision is millisecond instead of second), and the length of List/Hash/Set/ZSET/Stream can be also extended from 32 to 64 bits.
```

Comment:
```suggestion
- Allow to enable 64 bit expiration time (precision is in milliseconds instead of seconds), and the length of List/Hash/Set/ZSET/Stream can be also extended from 32 to 64 bits.
```